docs/user/exercises/quiz.rst

```diff
@@ -1,2 +1,440 @@
+
+        .. figure:: quiz/apollon-diagrams.png
+            :align: center
+
+    - On the Apollon Diagrams page, it is possible to see the list of Apollon Diagrams and the possible actions to perform with them.
+
+        - Open
+        - Delete
+
+    - Clicking on the creation of a new Apollon Diagram button |CreateANewApollonDiagram| opens the following form:
+
+        .. figure:: quiz/Apollon-form.png
+            :align: center
+            :scale: 50
+
+        - Title: provide the title of the Drag-and-Drop Model Quiz
+
+        - Diagram Type: It is a list picker to select between several diagrams:
+
+            - Class Diagram
+
+            - Activity Diagram
+
+            - Object Diagram
+
+            - Use Case Diagram
+
+            - Communication Diagram
+
+            - Component Diagram
+
+            - Deployment Diagram
+
+            - Petri Net
+
+            - Syntax Tree
+
+            - Flowchart
+
+        - Save button
+
+        - Cancel button
+
+    - Click the save button to open the Apollon editor
+
+        .. figure:: quiz/apollon-editor.png
+            :align: center
+
+        - Title: Allows to edit the tile of the diagram.
+
+        - Crop image to selection: Allows to download the current selection.
+
+        - Download button: To download the selection.
+
+        - Generate a quiz exercise button.
+
+        - Save button.
+
+        - Modeling field: The items for modeling the diagram will be displayed here.
+
+        - Elements to Drag and Drop in the Modeling field.
+
+Import a Quiz
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+    - In the quiz exercises section, click on the Import a Quiz button |ImportQuizButton|.
+
+    - The list of existing quizzes will appear.
+
+        .. figure:: quiz/import-list-quizzes.png
+            :align: center
+            :scale: 50
+
+        - The searching bar: Allows to look for a specific quiz by typing its name or part of it.
+
+        - The list of quizzes: Whit their ID, title, course and indicator if they are exam questions.
+
+        - Clicking the Import button |ImportButton| opens the quiz editor with the existing questions. Here it is possible to edit all parameters such as in **Create new quiz exercise**.
+
+Export a Quiz
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+    - In the quiz exercises section, click on the Export Quiz Exercises button |ExportQuizExerciseButton| and the list of quizzes will be shown
+
+        .. figure:: quiz/export-quizzes-list.png
+            :align: center
+            :scale: 50
+
+        - Select the quizzes for being exported in the Export Column.
+
+        - The Export button |ExportButton| will download the quiz in a JSON file.
+
+Resolve a Quiz Exercise
+--------
+
+    - If a quiz exercise is available on the Artemis Homepage as a current exercise, it will be possible to see it in the course overview or inside the course where it belongs.
```

Comment:
```suggestion
    - If a quiz exercise is available on the Artemis home page as a current exercise, it will be possible to see it in the course overview or inside the course where it belongs.
```
Under the section "Create new Quiz Exercise" you used home page -> so either use here home page or use Homepage in the section "Create new Quiz Exercise"